Testscripts/Linux/TVM-TestCompatibility.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+
+UtilsInit
+
+GetDistro
+case $DISTRO in
+    redhat*|centos*|oracle*)
+        echo "[packages-microsoft-com-azurecore]" | tee -a /etc/yum.repos.d/azurecore.repo
+        echo "name=packages-microsoft-com-azurecore" | tee -a /etc/yum.repos.d/azurecore.repo
+        echo "baseurl=https://packages.microsoft.com/yumrepos/azurecore/" | tee -a /etc/yum.repos.d/azurecore.repo
+        echo "enabled=1" | tee -a /etc/yum.repos.d/azurecore.repo
+        echo "gpgcheck=0" | tee -a /etc/yum.repos.d/azurecore.repo
+    ;;
+    ubuntu*|debian*)
+        release=$(lsb_release -c -s)
+        echo "deb [arch=amd64] http://packages.microsoft.com/repos/azurecore/ $release main" | sudo tee -a /etc/apt/sources.list.d/azure.list
+
+        wget -qO https://packages.microsoft.com/keys/microsoft.asc | apt-key add microsoft.asc
+        wget -qO https://packages.microsoft.com/keys/msopentech.asc | apt-key add msopentech.asc
+
```

Comment:
wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
 wget -qO - https://packages.microsoft.com/keys/msopentech.asc | apt-key add -

Reply:
Could you please run the commands?
```
wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add microsoft.asc
gpg: can't open 'microsoft.asc': No such file or directory
```